### ISSUE TITLE
NVDA/JAWS is announcing as 'Quick Input list' when keyboard focus is on edit field in command palette which is not descriptive enough. Place holder text should be read.

### ISSUE THREAD

**Issue by qpetraroia** [microsoft/vscode#178963]
Hello, we have been recently testing out our extension for accessibility bugs, and we believe we may have found one directly linked to vscode. The extension we are using is Bridge to Kubernetes.

https://learn.microsoft.com/en-us/visualstudio/bridge/bridge-to-kubernetes-sample

Does this issue occur when all extensions are disabled?: Yes

VS Code Version: Visual Studio Code Version 1.77.0. (user setup)
OS Version: Windows 11 Enterprise 22H2 (OS22621.1413)

Screen Reader:
NVDA and JAWS

Repro Steps:

1. Turn on NVDA/JAWS
2. Open Visual studio code application and setup Bridge to Kubernetes extension.
3. Download cluster config file and load 'stats-apo' project.
4. Press 'Ctrl+Shift+P' open command palette in View menu.
5. Select 'Bridge to Kubernetes: Configure' and press enter.
6. Check how NVDA is announcing when keyboard focus is on edit field in command palette.

User Impact: Screen reader users may not be able to understand that keyboard focus is on edit field if NVDA/JAWS announces as 'Quick Input list'. 

In the video below you can see that the "choose a service to redirect to your machine" is not read out and instead reads out the "quick input list" message.

https://user-images.githubusercontent.com/11465526/229407667-6f31582a-bf57-40d8-8291-8e8d82cf0fb5.mp4

Thanks!

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
